cmd/netpolicy/cmd/list.go

```diff
@@ -66,4 +72,6 @@ func init() {
 	// output options
 	listCmd.Flags().StringVarP(&focusWorkload, "focusworkload", "",
 		focusWorkload, "Focus connections of specified workload name in the output")
+	// output format - default txt
+	listCmd.PersistentFlags().StringVarP(&output, "output", "o", "txt", "Required output format (txt, json) (default value: txt)")
```

Comment:
I think it knows the default based on the passed in value of `txt` so no need to repeat in the option's explanation.